vm/build.rs

```diff
@@ -61,7 +58,12 @@ fn git(args: &[&str]) -> String {
     command("git", args)
 }
 
-fn command(cmd: &str, args: &[&str]) -> String {
+fn rustc_version() -> String {
+    let rustc = env::var_os("RUSTC").unwrap_or_else(|| "rustc".into());
+    command(rustc, &["-V"])
+}
```

Comment:
I agree about rand to reduce runtime efficiency, but do we really need to remove rustc_version? We only use it on build step and we don't need to take this extra code to maintain

Reply:
It also pulls in the `semver` crate to let you do semantic version matching, which we fully don't use at all. I do agree that a build dependency is not a very big deal, but the part of rust_version we use is basically exactly equivalent to this code, and we already have the `command` function here.

Reply:
I was actually gonna switch this to use `rustc_version::VersionMeta.short_version_string` instead of the plain 1.X.Y version (it's `rustc 1.85.0 (4d91de4e4 2025-02-17)` which is more similar to what pypy has: `[PyPy 7.3.18 with GCC 14.2.1 20250128]`) and then I realized that `short_version_string` is literally just the output of `rustc -V`, with no parsing of `rustc -Vv` needed at all.